### PR TITLE
Spanish translation updated

### PR DESCRIPTION
**Pull request by UbikaloPro**
I completed this translation taken care of all recommendations.
- Pull request to es.json 
- use of "usted" instead "tu"
- Checked for grammar 

original file in English at https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json

Note: this file is very different from previous es.json one, I copy the updated English file for this translation)

### CODE REVIEW

**Comment by mradkov** [comment]
> I completed this translation taken care of all recommendations.
> 
>     * Pull request to es.json
> 
>     * use of "usted" instead "tu"
> 
>     * Checked for grammar
> 
> 
> original file in English at https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json
> 
> Note: this file is very different from previous es.json one, I copy the updated English file for this translation)

Should we close https://github.com/aeternity/superhero-wallet/pull/414 then?

**Comment by mradkov** [comment]
> Can you also resolve the conflicts?

I guess I will resolve them since you did not @UbikaloPro.

**Comment by UbikaloPro** [comment]
Sorry I did not answered before. The es.json file in https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/es.json is not the updated file according with the one in https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json.
If there is a conflict between that es.json and the one I translated is because they are very different and their original en,json source is different. the only way I see is to replace it completely. 
There is some missing code at the end (in both files) I do not know what code should be there but I do know there is something missing in the last 10 lines (redirecting part)

**Comment by mradkov** [comment]
> Sorry I did not answered before. The es.json file in https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/es.json is not the updated file according with the one in https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json.
> If there is a conflict between that es.json and the one I translated is because they are very different and their original en,json source is different. the only way I see is to replace it completely.
> There is some missing code at the end (in both files) I do not know what code should be there but I do know there is something missing in the last 10 lines (redirecting part)

Yes. Please replace it completely - meaning make your translation based on the https://github.com/aeternity/superhero-wallet/blob/develop/src/popup/locales/en.json.

**Comment by UbikaloPro** [comment]
File checked and conflicts fixed

**Review by mradkov — APPROVED** [review]

File: src/popup/locales/es.json [hunk 1]
```diff
@@ -216,12 +283,12 @@
     "account": {
       "heading": "Cuenta",
       "accountName": "Nombre de la cuenta",
-      "copy": "copiar",
-      "send": "español",
+      "copy": "Copiar"
```
Comment (davidyuk) [inline src/popup/locales/es.json:286]: `,` is missed here

File: src/popup/locales/es.json [hunk 2]
```diff
@@ -203,9 +268,11 @@
       "mainAccount": "Cuenta principal",
       "reportBug": "Reportar un bug",
       "names": "Nombres",
-      "networks": "Networks",
-      "removeAccount": "eliminar cuenta",
+      "networks": "Redes",
+      "removeAccount": "Eliminar cuenta",
       "donateErrorReports": "Enviar reportes de errores"
```
Comment (davidyuk) [inline src/popup/locales/es.json:273]: `,` is missed here

File: src/popup/locales/es.json [hunk 3]
```diff
@@ -259,645 +326,656 @@
...
+      "cancelButton": "Denegar",
+      "confirmButton": "Aceptar",
+      "addressLabel": "Dirección",
+      "transactionLabel": "Transacción",
+      "addressRequest": "Esta aepp está solicitando acceso para ver tu dirección de cuenta actual",
+      "transactionRequest": "Esta aepp preparará una transacción. tu estás a cargo de confirmarla.",
+      "websiteRequestAccounts": "le gustaría obtener direcciones de cuenta",
+      "addressesLabel": "Direcciones",
+      "addressesRequest": "Esta aepp está solicitando acceso para ver las direcciones de tu cuenta"
+    }
```
Comment (davidyuk) [inline src/popup/locales/es.json:417]: `,` is missed here

File: src/popup/locales/es.json (hunk 1 above)
Comment (mradkov) [reply src/popup/locales/es.json:286]: Strange, the tests passed.

Comment (UbikaloPro) [reply src/popup/locales/es.json:286]: In all .json originals (superhero wallet and in-app ones) there is no "," in the last code line before "}".
example:
    "TipCommentsView": {
      "NoResultsMsg": "There are no comments at the moment.",
      "Reply": "Reply",
      "AddReply": "Add reply",
      "LatestReplies": "Latest Replies"
    },
    "SingleCommentView": {
      "NoResultsMsg": "There are no comments to display"
    },

As it is showed there the code lines before "}" have no ","

File: src/popup/locales/es.json (hunk 2 above)
Comment (UbikaloPro) [reply src/popup/locales/es.json:273]: same explanation as before....

File: src/popup/locales/es.json (hunk 3 above)
Comment (UbikaloPro) [reply src/popup/locales/es.json:417]: same explanation about "," as before

File: src/popup/locales/es.json (hunk 1 above)
Comment (davidyuk) [reply src/popup/locales/es.json:286]: > In all .json originals (superhero wallet and in-app ones) there is no "," in the last code line before "}".


It is true, is required by JSON format. But in this case the `,` is missed between the fields, like:
```
  ...
  "fieldName1": "fieldValue1"
  "fieldName2": "fieldValue2"
}
```
it should be
```
  ...
  "fieldName1": "fieldValue1",
  "fieldName2": "fieldValue2"
}
```

Also, GitHub highlights this typo with the red color:

Screenshot 2020-08-24 at 15 29 14

But don't worry, anyway, we have fixed it in #476

**Pull request merged**